CHANGELOG.md

```diff
@@ -53,6 +53,8 @@
 - Removed a hardcoded proxy-wasm isolation level setting that was preventing the
   `nginx_http_proxy_wasm_isolation` configuration value from taking effect.
   [#11407](https://github.com/Kong/kong/pull/11407)
+- Fix TTL not working in DB-less and Hybrid mode.
```

Comment:
With new change log format we should remove this change.

Reply:
hi @chronolaw , do you mean we only need to introduce [CHANGELOG/unreleased/kong/11464.yaml](https://github.com/Kong/kong/pull/11464/files#diff-9acd13b296709af1041ac3525c960872919b8a546f307d97665311d3c46df49b), not modifying CHANGELOG.md?

Reply:
Yes, you are right.

Reply:
done